src/components/modules/epics/ContributionsEpic.stories.tsx

```diff
@@ -112,7 +112,10 @@ WithAboveArticleCount.args = {
             type: 'above',
         },
     },
-    numArticles: 99,
+    articleCounts: {
+        total: 99,
+        forTargetedWeeks: 99,
```

Comment:
I feel like there are maybe better names for these, but I can't think of any 😂 `total` = past 52 weeks, and `forTargetedWeeks` = past x weeks, as configured by the tool.

Reply:
maybe if their names matched up a little more?

```ts
interface ArticleCounts {
  for52Weeks: number;
  forTargetedWeeks: number;
}
```

Reply:
maybe `Targeted` -> `Configured` ?

Reply:
I'm really not sure haha!

Reply:
Yes it's difficult to name them, and the fact I had to write comments next to each in the type definition is a sign that they're poorly named!
For `total`, it's up to 52 weeks (I don't know if that number will ever change) but depends how far back the browser's history goes.
`forTargetedWeeks` is a weird name but I wanted to emphasise that it's linked to the targeting configuration

Reply:
Update - I went with `for52Weeks` instead of `total`